### PR TITLE
USWDS-Compile - Dependencies: Use Node LTS 20

### LINKED ISSUE #71: USWDS-Compile - Feature: Update USWDS version & other deps
### Is your feature request related to a problem? Please describe.

Update dependencies for 1.1.0 milestone:

- @uswds/uswds 
- autoprefixer 
- sass-embedded

### Describe the solution you'd like

Update USWDS & other dependencies.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-compile/issues) for duplicate feature requests.

### PR DESCRIPTION
**Pull request by mejiaj**
**Use latest Node LTS 20 and update dependencies.** Using latest LTS version of node and updated version of USWDS and SASS. Closes #71.

## Dependencies

| Dependency    | Old     | New     |
| ------------- | ------- | ------- |
| @uswds/uswds  | 3.5.0   | 3.7.0   |
| autoprefixer  | 10.4.14 | 10.4.16 |
| sass-embedded | 1.63.6  | 1.69.5  |

## How to test

1. Go to USWDS-Sandbox repo.
2. Copy `test-uswds-compile-78-node-update` branch.
3. Run`npm install`.
4. Run tasks (there should be no regressions):
    - `npm run uswds:copyAssets`
    - `npm run watch`

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 20.9.0
```
Comment (thisisdano) [inline .tool-versions:1]: Added a `.tool-versions` file for folks who use other version managers like `asdf`

**Review by thisisdano — APPROVED** [review]
LGTM

**Review by amyleadem — APPROVED** [review]
Looks good to me. Here is what I tested: 

- [x] Successfully ran the `uswds-sandbox` tests described in the PR description:
    - [x] Ran `npm install`, `npm run uswds:copyAssets`, `npm run watch` without error
    - [x] Confirmed that styles render as expected
- [x] Checked that references to previous node versions in docs have been updated to v20
- [x] Confirmed README formatting hasn’t changed

**Review by mahoneycm — APPROVED** [review]
Looks good to me!

- [x] Installation and tasks run without errorr
- [x] Testing on sandbox shows no regressions
- [x] Read me page looks good

**Pull request merged**